reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -133,7 +133,10 @@ def _format_arguments(self):
                  --outputfile \"results/greetings.txt\" --sleeptime 0
         """
         if self.workflow.type_ == 'serial':
-            arguments = re.sub(r'"', '\\"', " ".join(self.cmd[2].split()[3:]))
+            raw_cmd = " ".join(self.cmd[2].split()[3:])
+            encoded_raw_cmd = base64.b64encode(
+                raw_cmd.encode('utf-8')).decode('utf-8')
+            arguments = 'echo {}|base64 -d|bash'.format(encoded_raw_cmd)
```

Comment:
Works nicely :+1: The only comment is that the user logs now show:

```console
$ reana-client logs -w htc
2020-01-08T17:44:56 INFO     Parameters: inputfile=data/names.txt outputfile=results/greetings.txt sleeptime=0.0
command to execute: echo cHl0aG9uICJjb2RlL2hlbGxvd29ybGQucHkiIC0taW5wdXRmaWxlICJkYXRhL25hbWVzLnR4dCIgLS1vdXRwdXRmaWxlICJyZXN1bHRzL2dyZWV0aW5ncy50eHQiIC0tc2xlZXB0aW1lIDA=|base64 -d|bash
```

It would be nicer to decode the string for people (i.e. run `echo ... | base64 -d` part without final `bash`) so that the logs would say:

```
command to execute:
python "code/helloworld.py" --inputfile "data/names.txt" --outputfile "results/greetings.txt" --sleeptime 0
```

because the encoding is just a "transport mechanism" for the logic, so to speak, and the users would rather see the logic.

Can we cook something up easily for all of CWL/Serial/Yadage?